### PR TITLE
Update display tokens page

### LINKED ISSUE #1311: Update display tokens page to include token autodetection
Update intro paragraph on Display tokens page to note that users can also turn on autodetection for ERC-20 tokens.

### PR DESCRIPTION
**Pull request by alexandratran**
# Description

Update description of Display tokens page to mention token autodetection.

## Issue(s) fixed

Fixes #1311

## Preview

https://docs.metamask.io/1311-update-display-tokens/wallet/how-to/display/tokens/

## Checklist

Complete this checklist before merging your PR:

- [ ] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page.
- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

File: wallet/how-to/display/tokens.md [hunk 1]
```diff
@@ -19,11 +19,14 @@ You can improve the security and experience of displaying your
 `wallet_watchAsset` provides a friendly interface that prompts users to register tokens to their
 MetaMask wallet, without having to interact with contract addresses.
 
-:::tip Displaying NFTs
-With `wallet_watchAsset`, you can prompt users to add their NFTs even when they have NFT
-autodetection disabled.
-Moreover, NFT autodetection only detects NFTs on Ethereum Mainnet.
-With `wallet_watchAsset`, users can add NFTs from other networks.
+:::tip
+With `wallet_watchAsset`, you can prompt users to add their ERC-20 tokens and NFTs even when they
```
Comment (joaniefromtheblock) [inline wallet/how-to/display/tokens.md:23]: I think this a passive construction. Maybe...
```suggestion
"Use `wallet_watchAsse`t to prompt users to add their ERC-20 tokens and NFTs even when token autodetection or NFT autodetection is disabled."
```

File: wallet/how-to/display/tokens.md [hunk 2]
```diff
@@ -19,11 +19,14 @@ You can improve the security and experience of displaying your
...
+have token autodetection or NFT autodetection disabled.
+
+Moreover, token autodetection only detects ERC-20 tokens on
```
Comment (joaniefromtheblock) [inline wallet/how-to/display/tokens.md:26]: Moreover might be extraneous. Maybe remove for brevity?
```suggestion
Token autodetection only detects ERC-20 tokens on
```

**Review by joaniefromtheblock — APPROVED** [review]
LGTM. Just a minor suggestion

File: wallet/how-to/display/tokens.md (hunk 1 above)
Comment (alexandratran) [reply wallet/how-to/display/tokens.md:23]: The intention of this tip is to expand on the benefits of using `wallet_watchAsset`. It's not necessarily an instruction, hence the wording.

File: wallet/how-to/display/tokens.md (hunk 2 above)
Comment (alexandratran) [reply wallet/how-to/display/tokens.md:26]: It's meant to emphasize an additional benefit. Would "also" be better?

**Pull request merged**